tests/hpc/slurm_master_db.pm

```diff
@@ -53,39 +212,29 @@ sub run {
     barrier_wait("SLURM_SLAVE_SERVICE_ENABLED");
 
     $self->check_nodes_availability();
-    assert_script_run("srun -w slave-node00 date");
 
-    ## TODO provide more comprehensive checks for slurm
-    # job scheduling
-    for (my $i = 0; $i < 20; $i++) {
-        assert_script_run("srun -w slave-node00 date");
-        assert_script_run("srun -w slave-node01 date");
-    }
+    my $file = 'tmpresults.xml';
+    assert_script_run("touch $file");
+    script_run("echo \"<testsuite name='HPC single tests'>\" | tee $file");
```

Comment:
I would really consider using something like https://metacpan.org/pod/XML::Writer , otherwise you will endup with hardly maintainable and readable code

Reply:
yup! this is meant to simply work for now. With external parser for HPC tests I will surely use your suggestion.